src/components/new-request/name-check/name-check.vue

```diff
@@ -0,0 +1,809 @@
+                      {{ expandBtnTxt }}
+                    </a>
+                  </p>
+                  <div v-if="expandHelpTxt">
+                    <p class="ma-0 pt-7">
+                      The Similar Name Check searches the Corporate Register only;
+                      it may not find ALL similarly named businesses. This register
+                      includes the names of corporations incorporated or registered
+                      extraprovincially in British Columbia. It does not include names
+                      of British Columbia firms, trademarks or corporations registered
+                      outside British Columbia.
+                    </p>
+                    <p class="ma-0 pt-7">
+                      Do not commit to any name before it is reviewed.
+                    </p>
+                    <p class="ma-0 pt-7">
+                      If you want to ensure your name is not used outside of British
+                      Columbia, you could also access the Trademarks database at
+                      <a class="txt-link" :href="tradeMarkDBLink" target="_blank">
+                        <span>www.strategis.ic.gc.ca</span>
+                      </a>
+                      &nbsp;
+                      <a :href="tradeMarkDBLink" style="text-decoration: none;" target="_blank">
+                        <v-icon color="primary" small>mdi-open-in-new</v-icon>
+                      </a>
+                      , or you may wish to search other
+                      jurisdictions in Canada. Most public business and trademark
+                      registers in Canada are reflected in the NUANS database, which may
+                      be searched for a fee through private search firms.
+                    </p>
+                    <p class="ma-0 pt-7">
+                      <u>You should also research your desired name by searching the internet,
+                      social media, and relevant internet domain names if these are
+                      considerations for your business.</u>
+                    </p>
+                  </div>
+                </div>
+              </v-col>
+            </v-row>
+            <NameCheckConflicts :items="itemsConflict" @clear-error="clearError" @retry="retry"/>
+          </v-tab-item>
+        </v-tabs-items>
+      </v-container>
+      <v-row no-gutters class="pt-7 name-check-info-text-no-border">
+        <v-col cols="auto">
+          <v-icon>mdi-information-outline</v-icon>
+        </v-col>
+        <v-col class="pl-2">
+          <span>{{ bottomText }}</span>
+        </v-col>
+      </v-row>
+      <v-row no-gutters class="pa-10 pt-7">
+        <v-col>
+          <v-btn @click="back()" class="outlined" outlined>
+            Start Over
+          </v-btn>
+        </v-col>
+        <v-col>
+          <v-btn id="name-check-submit-btn" style="float: right;" @click="dialogCheck()">
```

Comment:
Try Vuetify class "float-right" instead of inline style.

https://vuetifyjs.com/en/styles/float/#classes

Reply:
I knew there was something easier